9.1.Индексация/9.1.6.py

```diff
@@ -0,0 +1,3 @@
+s = "In 2010, someone paid 10k Bitcoin for two pizzas."
+
+print(s[39])
```

Comment:
Можно так, конечно, но гораздо проще в обратную сторону:
```suggestion
print(s[-10])
```